artisatomic/__init__.py

```diff
@@ -1266,7 +1285,13 @@ def interpret_configuration(instr_orig: str) -> tuple[list[str], int, int, int,
         term_parity = 0  # even
     else:
         term_parity = [0, 1][(instr[-1] == "o")]
-        instr = instr[:-1]
+        if not any(char in lchars for char in instr):
+            # This will be an incorrectly formatted QUB file with no term
+            print("Warning: Check QUB file formatting")
+            instr = instr
+        else:
+            # Preserve previous behaviour
+            instr = instr[:-1]
```

Comment:
I think you stumbled on some unused code that was intended for CMFGEN data, but if it was useful to you, then ok!